package.json

```
@@ -6,26 +6,76 @@
   "description": "Tests for the New Relic JavaScript agent",
   "license": "Apache-2.0",
   "homepage": "https://docs.newrelic.com/docs/browser/browser-monitoring",
-  "main": "",
+  "main": "./dist/cjs/index.js",
   "types": "./types.ts",
-  "module": "./dist/mjs/index.mjs",
+  "module": "./dist/esm/index.js",
   "exports": {
     ".": {
-      "import": "./dist/mjs/index.mjs",
-      "require": "./dist/cjs/index.cjs"
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js"
+    },
+    "./custom-agent": {
+      "import": "./dist/esm/custom-agent.js",
+      "require": "./dist/cjs/custom-agent.js"
+    },
+    "./browser-agent": {
+      "import": "./dist/esm/loaders/browser-agent.js",
+      "require": "./dist/cjs/loaders/browser-agent.js"
+    },
+    "./micro-agent": {
+      "import": "./dist/esm/loaders/micro-agent.js",
+      "require": "./dist/cjs/loaders/micro-agent.js"
+    },
+    "./worker-agent": {
+      "import": "./dist/esm/loaders/worker-agent.js",
+      "require": "./dist/cjs/loaders/worker-agent.js"
+    },
+    "./features/ajax": {
+      "import": "./dist/esm/features/ajax/index.js",
+      "require": "./dist/cjs/features/ajax/index.js"
+    },
+    "./features/jserrors": {
+      "import": "./dist/esm/features/jserrors/index.js",
+      "require": "./dist/cjs/features/jserrors/index.js"
+    },
+    "./features/metrics": {
+      "import": "./dist/esm/features/metrics/index.js",
+      "require": "./dist/cjs/features/metrics/index.js"
+    },
+    "./features/page_action": {
+      "import": "./dist/esm/features/page_action/index.js",
+      "require": "./dist/cjs/features/page_action/index.js"
+    },
+    "./features/page_view_event": {
+      "import": "./dist/esm/features/page_view_event/index.js",
+      "require": "./dist/cjs/features/page_view_event/index.js"
+    },
+    "./features/page_view_timing": {
+      "import": "./dist/esm/features/page_view_timing/index.js",
+      "require": "./dist/cjs/features/page_view_timing/index.js"
+    },
+    "./features/session_trace": {
+      "import": "./dist/esm/features/session_trace/index.js",
+      "require": "./dist/cjs/features/session_trace/index.js"
+    },
+    "./features/spa": {
+      "import": "./dist/esm/features/spa/index.js",
+      "require": "./dist/cjs/features/spa/index.js"
     }
   },
+  "engines": {
+    "node": ">=12.16.0 < 13.0.0 || >=13.7.0"
```

This should be `>=12.17.0` https://nodejs.org/api/packages.html#nodejs-packagejson-field-definitions
```suggestion
    "node": ">=12.17.0 < 13.0.0 || >=13.7.0"
```